src/features/ShopHomePage.js

```diff
@@ -11,7 +15,36 @@ export default function ShopHomePage() {
   const { data: products, isLoading: isLoadingProducts } =
     useGetProductsQuery(productBatch);
 
-  if (isLoadingCategories || isLoadingProducts) {
+  const { data: bestRatedProducts, isLoading: isLoadingBestRated } =
+    useGetAllProductsQuery(undefined, {
+      selectFromResult: ({ data, isLoading, isUninitialized }) => {
+        const filteredData = (data ?? []).filter(
+          (product) => product.rating > 4.9
+        );
+        return {
+          data: filteredData,
+          isLoading: isUninitialized || isLoading,
+        };
+      },
+    });
+
+  console.log(bestRatedProducts);
```

Comment:
remove console.log and commented dummycode below

Reply:
done! tysm.